src/Toggl2Vertec/Vertec6/Requests/GetEmploymentFactor.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Toggl2Vertec.Vertec6.Api;
+
+namespace Toggl2Vertec.Vertec6.Requests;
+
+/// <summary>
+/// Returns the part-time employment factor of a person on a given date as a value between 0 and 1,
```

Comment:
@adwicki : Speculation on my part based on how you are using the value in the code

Reply:
@lord-executor Absolutely correct, thats also what I _assume_ the API does...